### DISCUSSION TITLE
RetroArch No Sound & Low Frame rate Upon Resuming Session

### DISCUSSION THREAD

**Discussion by QuirkyKirkHax** [LizardByte/community#176] [Sunshine] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your issue described in the documentation?

- [X] I have read the documentation

### Is your issue present in the nightly release?

- [X] This issue is present in the nightly release

### Describe the Bug

I've got a weird issue where if I launch a game on RetroArch through Bigbox, then quit Moonlight, and resume the session, there is no sound and the frame rate tanks, like to 5fps or so.

I can quit RetroArch within the session and everything is fine, but RetroArch doesn't seem to handle the resume well.  I'm assuming it's Sunshine, I don't recall this happening on Gamestream

This appears to happen exclusively on RetroArch though.  Other emulators and games work no issue.  Tested this in both Fullscreen and borderless window mode.

### Expected Behavior

I should be able to resume a session without any degradation in video/frame rate.  At least, no more than a few seconds.

### Host Operating System

Windows

### Operating System Version

10 22H2

### Architecture

None

### Sunshine commit or version

0.18.3

### Package

Windows - installer

### GPU Type

Nvidia

### GPU Model

RTX 3070 FE

### GPU Driver/Mesa Version

528.49

**Comment by ReenigneArcher** [comment, marked as answer]
I suspect your selected audio device is being changed for some reason.

**Reply by QuirkyKirkHax** [reply]
I can't imagine why.  I've got an HDMI dummy plug in the server, but no other audio device.  Unless it's some emulated audio device from RDP, but there's no connected RDP session when this issue occurs.

Not sure what other audio devices it could be.

**Reply by QuirkyKirkHax** [reply]
I managed to fix this issue this week by changing the audio driver within retroarch from xaudio to dsound.  Not sure why that worked, but here is a short video showing the error and the fix.  Unsure if this is something on Sunshine's side or not, as this never happened with gamestream.

https://youtu.be/0jKz35tlci8
